packages/autocomplete-js/src/types/AutocompleteRenderer.ts

```diff
@@ -33,4 +39,13 @@ export type AutocompleteRenderer = {
    * @default preact.Fragment
    */
   Fragment: PragmaFrag;
+  /**
+   * The function to render children to an element.
+   */
+  render?: Render;
 };
+
+export type HTMLTemplate = (
+  strings: TemplateStringsArray,
+  ...values: any[]
+) => VNode | VNode[];
```

Comment:
I couldn't type it as:

```tsx
export type HTMLTemplate = ReturnType<typeof htm.bind<VNode>>
```

You can't pass a generic type when using `typeof` like this.

Reply:
I'm ok with the type redone, it will still fail if they don't match on usage with html

Reply:
Yep, not ideal but will work for now. We can PR in `htm` later to have a better type, if possible.